README-install.md

```diff
@@ -119,16 +119,16 @@ openshift-ansible-callback-plugins openshift-ansible-filter-plugins
 openshift-ansible-lookup-plugins openshift-ansible-playbooks
 openshift-ansible-roles
 
-    # yum install openshift-ansible \
-      openshift-ansible-callback-plugins openshift-ansible-filter-plugins \
-      openshift-ansible-lookup-plugins openshift-ansible-playbooks \
-      openshift-ansible-roles
+        # yum install openshift-ansible \
+          openshift-ansible-callback-plugins openshift-ansible-filter-plugins \
+          openshift-ansible-lookup-plugins openshift-ansible-playbooks \
+          openshift-ansible-roles
```

Comment:
these look ok now, and do not appear to need extra indentation: https://github.com/ViaQ/Main/blob/master/README-install.md#installing-viaq-packages